src/ReactNativeImports.ts

```diff
@@ -0,0 +1,70 @@
+const ReactNative = {
+  Animated: { createAnimatedComponent: (a: any) => a },
+  Easing: null,
+  processColor: null,
+  StyleSheet: { create: (a: any) => a },
+  StyleProp: null,
+  ViewStyle: null,
+  View: null,
+  TouchableHighlightProps: null,
+  TouchableOpacityProps: null,
+  ColorValue: null,
+  ViewProps: null,
+  TouchableWithoutFeedbackProps: null,
+  Insets: null,
+  TouchableNativeFeedback: null,
+  ScrollView: null,
+  ScrollViewProps: null,
+  Switch: null,
+  SwitchProps: null,
+  TextInput: null,
+  TextInputProps: null,
+  DrawerLayoutAndroid: null,
+  DrawerLayoutAndroidProps: null,
+  FlatList: null,
+  FlatListProps: null,
+  RefreshControl: null,
+  I18nManager: { isRTL: false },
+  LayoutChangeEvent: null,
+  Keyboard: null,
+  StatusBar: null,
+  StatusBarAnimation: null,
+  NativeSyntheticEvent: null,
+  PixelRatio: { get: () => window.devicePixelRatio },
+};
+
+export const Animated = ReactNative.Animated;
+export const Easing = ReactNative.Easing;
+export const processColor = ReactNative.processColor;
+export const StyleSheet = ReactNative.StyleSheet;
+export type StyleProp = typeof ReactNative.StyleProp;
+export type ViewStyle = typeof ReactNative.ViewStyle;
+export const View = ReactNative.View;
+export type TouchableHighlightProps =
+  typeof ReactNative.TouchableHighlightProps;
+export type TouchableOpacityProps = typeof ReactNative.TouchableOpacityProps;
+export type ColorValue = typeof ReactNative.ColorValue;
+export type ViewProps = typeof ReactNative.ViewProps;
+export const TouchableWithoutFeedbackProps =
+  ReactNative.TouchableWithoutFeedbackProps;
+export const Insets = ReactNative.Insets;
+export const TouchableNativeFeedback = ReactNative.TouchableNativeFeedback;
+export const ScrollView = ReactNative.ScrollView;
+export type ScrollViewProps = typeof ReactNative.ScrollViewProps;
+export const Switch = ReactNative.Switch;
+export type SwitchProps = typeof ReactNative.SwitchProps;
+export const TextInput = ReactNative.TextInput;
+export type TextInputProps = typeof ReactNative.TextInputProps;
+export const DrawerLayoutAndroid = ReactNative.DrawerLayoutAndroid;
+export type DrawerLayoutAndroidProps =
+  typeof ReactNative.DrawerLayoutAndroidProps;
+export const FlatList = ReactNative.FlatList;
+export type FlatListProps = typeof ReactNative.FlatListProps;
+export const RefreshControl = ReactNative.RefreshControl;
+export const I18nManager = ReactNative.I18nManager;
+export const LayoutChangeEvent = ReactNative.LayoutChangeEvent;
+export const Keyboard = ReactNative.Keyboard;
+export const StatusBar = ReactNative.StatusBar;
+export const StatusBarAnimation = ReactNative.StatusBarAnimation;
+export const NativeSyntheticEvent = ReactNative.NativeSyntheticEvent;
+export const PixelRatio = ReactNative.PixelRatio;
```

Comment:
Doesn't `export default ReactNative` do the same thing?

Reply:
[35a8aaa](https://github.com/software-mansion/react-native-gesture-handler/pull/2852/commits/35a8aaa5a1a0ec1ddfc15dc81a5eebba069cd37f)

![image](https://github.com/software-mansion/react-native-gesture-handler/assets/63123542/4aa80532-c0e5-4117-8287-6998fdb771d6)

Reply:
We can try `export const { Animated, Easing } = ReactNative;` syntax

Reply:
Even though we were talking about different things, you've inspired us to change those exports to yet another form ([00c0d6c](https://github.com/software-mansion/react-native-gesture-handler/pull/2852/commits/00c0d6cff2c825950254744aad374efedf33af9e))